tests/e2e.sh

```diff
@@ -145,7 +145,18 @@ assert_dir_exists knative-serving
 # kpt fn Tests
 ############################
 
-testcase "kpt_set_namespace_success"
+testcase "kpt_set_namespace_go_docker_success"
+kpt pkg get $SDK_REPO/example-configs example-configs
+kpt fn run . --image gcr.io/kpt-functions/set-namespace -- namespace=example-ns
+assert_contains_string example-configs/gatekeeper.yaml "namespace: example-ns"
+
+testcase "kpt_set_namespace_go_success"
+kpt pkg get $SDK_REPO/example-configs example-configs
+kpt pkg get $CATALOG_REPO/functions/go ./
+kpt fn run . --enable-exec --exec-path go/set_namespace/set_namespace -- namespace=example-ns
```

Comment:
Replace this command with `kpt fn run example-configs --enable-exec --exec-path "$(pwd)"/go/set-namespace/set-namespace -- namespace=example-ns`

Reply:
Done